.stylelintrc.json

```
@@ -0,0 +1,9 @@
+{
+  "extends": "stylelint-config-recommended-scss",
+  "rules": {
+    "scss/no-global-function-names": null,
+    "no-descending-specificity": null,
+    "scss/comment-no-empty": null,
+    "selector-class-pattern": "^[a-z]([a-z0-9-]+)?(__([a-z0-9]+-?)+)?(--([a-z0-9]+-?)+){0,2}$"
```

JSON doesn't do comments, but this is the best I could find to match the hyphenated BEM pattern rule that we had previously

---

Could maybe simplify it a bit? Dropping capture groups, not allowing multiple modifiers (not valid anyways?).

```suggestion
    "selector-class-pattern": "^[a-z][a-z0-9-]*(__[a-z0-9-]+)?(--[a-z0-9-]+)?$"
```

https://regex101.com/r/EXsyIg/1